fuzz/src/chanmon_consistency.rs

```diff
@@ -1687,6 +1703,19 @@ pub fn do_test<Out: Output>(data: &[u8], underlying_out: Out, anchors: bool) {
 				nodes[2].maybe_update_chan_fees();
 			},
 
+			0xa0 => {
+				nodes[0].maybe_propose_quiescence(&nodes[1].get_our_node_id(), &chan_a_id).unwrap()
+			},
+			0xa1 => {
+				nodes[1].maybe_propose_quiescence(&nodes[0].get_our_node_id(), &chan_a_id).unwrap()
+			},
+			0xa2 => {
+				nodes[1].maybe_propose_quiescence(&nodes[2].get_our_node_id(), &chan_b_id).unwrap()
+			},
+			0xa3 => {
+				nodes[2].maybe_propose_quiescence(&nodes[1].get_our_node_id(), &chan_b_id).unwrap()
+			},
```

Comment:
Not sure how much it matters, but we could do `nodes[2]` to `nodes[0]`, too, and vice versa.

Reply:
Ah, ok. Was thinking about our functional test setup.